### PR TITLE
blog: draft serve/update status blog post

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu,

Please review the following commits I made in branch nicks/ch10508:

bee6f456ed270769132f336862aa9fff9048066d (2020-12-14 15:23:01 -0500)
blog: draft serve/update status blog post

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
this is the draft we paired on together, feel free to push your own changes!

File: blog/_posts/2020-12-15-what-color-is-your-status.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+slug: "what-color-is-your-status"
+date: 2020-12-14
+author: han
+layout: blog
+title: "What Color is Your Status"
```
Comment (nicks) [inline blog/_posts/2020-12-15-what-color-is-your-status.md:6]: idea: do you know how many services are in your dev environment?

**Comment by victorwuky** [comment]
@nicks @hyu : The app uses an `x`, a checkmark, and tooltips, in addition to colors. Would it be worth it to mention these in the context of accessibility? I worry if the blog post talks about colors (especially if that's the title of the post), readers might mistakenly think we haven't considered accessibility, which I know we have, from the conversations I've seen, and that it's already implemented. Or maybe we don't have to mention this in the text of the blog post at all, and just show screenshots indicating that the design is accessible.

Screen Shot 2020-12-14 at 4 40 16 PM

**Comment by victorwuky** [comment]
And if you do want to mention accessbility in the post, might be a good idea to link to https://blog.tilt.dev/2020/04/02/march-2020-commit-of-the-month.html as prior work.

**Comment by nicks** [comment]
ya, that's true. this is just a first pass, certainly not the final version.

**Comment by hyu** [comment]
@nicks — I kept the same structure we paired on, but made the copy more conversational and tried a new title that doesn't mention color. (Good points, @victorwuky!) 

Thoughts?

Still need an illustration / screenshot, but...thought I'd run this revision by you first.

**Comment by nicks** [comment]
too real?

3tayif

**Comment by nicks** [comment]
love it, let's slap an image on it and ship it!

**Comment by nicks** [comment]
Screenshot from 2020-12-16 10-39-29

@hyu here's what I added, do you want to do the honor of merging??

**Pull request merged**